shared/src/commonMain/kotlin/db/repository/level/RemoteLevelRepository.kt

```diff
@@ -34,19 +36,21 @@ class RemoteLevelRepository :
     override suspend fun fetchById(id: String): Level {
         try {
             val levelDocument =
-                firestore.collection("LEVELS").document(id).get()
+                firestore.collection(FIREBASE_LEVEL_COLLECTION).document(id).get()
             return levelDocument.data()
         } catch(e: Exception) {
             throw e
         }
     }
 
     override suspend fun update(id: String, request: CreateLevelRequest) {
-        firestore.collection("LEVELS").document(id).update(request)
+        firestore.collection(FIREBASE_LEVEL_COLLECTION).document(id).update(request)
+        database.update(id, request)
     }
 
     override suspend fun delete(id: String) {
-        firestore.collection("LEVELS").document(id).delete()
+        firestore.collection(FIREBASE_LEVEL_COLLECTION).document(id).delete()
```

Comment:
What happens when this delete fails? Will the local delete be executed? The local db changes can only happen after the networking calls succeed